packages/backend/src/graphql/resolvers/outgoing_payment.ts

```diff
@@ -119,6 +118,28 @@ export const requoteOutgoingPayment: MutationResolvers<ApolloContext>['requoteOu
     }))
 }
 
+export const sendOutgoingPayment: MutationResolvers<ApolloContext>['sendOutgoingPayment'] = async (
+  parent,
+  args,
+  ctx
+): ResolversTypes['OutgoingPaymentResponse'] => {
```

Comment:
If the payment state is not `Ready` should that be a client (400) error?

Reply:
Yeah, and maybe the same for `requoteOutgoingPayment` and `cancelOutgoingPayment`.

Should we define a `WrongPaymentState` error that the resolvers can catch and return `400`?

Reply:
Yeah. The alternative would be to check the state in the resolver itself, but I think that validation logic is better off in the service (where it already is).